_data/toc/module-reference-guide-2_3.yml

```diff
@@ -1,7 +1,8 @@
-label: Magento Reference Guide
+label: Module Reference Guide
 pages:
-  - label: Introduction
+  - label: INTRODUCTION
```

Comment:
We don't want this appearing as all caps in the HTML.

Reply:
fixed